Source/Diagnostics/FieldIO.cpp

```diff
@@ -636,9 +636,9 @@ WarpX::AverageAndPackFields ( Vector<std::string>& varnames,
 #pragma omp parallel
 #endif
                 for (MFIter mfi(mf_avg[lev]); mfi.isValid(); ++mfi) {
-                    (mf_avg[lev])[mfi].setVal(static_cast<Real>(ParallelDescriptor::MyProc()),
-                                              dcomp++);
+                    (mf_avg[lev])[mfi].setVal(static_cast<Real>(ParallelDescriptor::MyProc()));
                 }
+                dcomp++;
```

Comment:
@WeiqunZhang @atmyers  Can I move
 (mf_avg[lev])[mfi].setVal(static_cast<Real>(ParallelDescriptor::MyProc())) 
outside the MFIter loop and instead do the following:
mf_avg[lev].setVal(static_cast<Real>(ParallelDescriptor::MyProc()));